### PR TITLE
add symbols map for pattern character

### PR DESCRIPTION
**Pull request by gufengwyx8**

### CODE REVIEW

File: src/main/java/ru/lanwen/verbalregex/VerbalExpression.java [hunk 1]
```diff
@@ -18,6 +20,19 @@
         private StringBuilder suffixes = new StringBuilder();
         private int modifiers = Pattern.MULTILINE;
 
+        private static final Map<Character, Integer> SYMBOL_MAP;
```
Comment (lanwen) [inline src/main/java/ru/lanwen/verbalregex/VerbalExpression.java]: I'd prefer initializer 
```java
 private static final Map<Character, Integer> SYMBOL_MAP = new HashMap<Character, Integer>() {{
        put(...);
        put(...);
    }} ;
```

which is more readable imho

**Comment by gufengwyx8** [comment]
Thank you for your advice!

**Comment by gufengwyx8** [comment]
Thank you for your advice!

**Comment by gufengwyx8** [comment]
Thank you for your advice!

**Comment by gufengwyx8** [comment]
Thank you for your advice!

**Comment by gufengwyx8** [comment]
Thank you for your advice!

**Comment by gufengwyx8** [comment]
Thank you for your advice!

**Comment by lanwen** [comment]
thanks!

**Pull request merged**